plenum/server/consensus/view_change_service.py

```diff
@@ -80,29 +80,29 @@ def process_need_view_change(self, msg: NeedViewChange):
         # 3. Update shared data
         self._data.view_no = view_no
         self._data.waiting_for_new_view = True
+        self._data.primary_name = None
         if not self._data.is_master:
-            self._data._master_reordered_after_vc = False
-        self._data.primaries = self._primaries_selector.select_primaries(view_no=self._data.view_no)
-        for i, primary_name in enumerate(self._data.primaries):
-            logger.display("{} selected primary {} for instance {} (view {})"
-                           .format(PRIMARY_SELECTION_PREFIX,
-                                   primary_name, i, self._data.view_no),
-                           extra={"cli": "ANNOUNCE",
-                                  "tags": ["node-election"]})
-
-        old_primary = self._data.primary_name
-        self._data.primary_name = generateName(self._data.primaries[self._data.inst_id], self._data.inst_id)
+            self._data.master_reordered_after_vc = False
 
         if not self._data.is_master:
             return
```

Comment:
I think it is possible to get rid of second `if not is_master` and put `return` under previous one

Reply:
Done